pkg/provider/kubernetes/settings.go

```diff
@@ -99,6 +99,10 @@ func (s *SettingsProvider) createDefaultGlobalSettings() (*kubermaticv1.Kubermat
 				MaxRAM:    128,
 				EnableGPU: false,
 			},
+			OpaOptions: kubermaticv1.OpaOptions{
```

Comment:
Nit: actually this is not needed to add it here, as the field is not a pointer and the default values are false anyway ;)